tflog/subsystem_test.go

```diff
@@ -31,7 +35,7 @@ func TestSubsystemWith(t *testing.T) {
 				{
 					"@level":        hclog.Trace.String(),
 					"@message":      "test message",
-					"@module":       testSubsystem,
+					"@module":       testSubsystemModule,
```

Comment:
And for all of the `@module` in this test file

Reply:
Updated the constant to include the hardcoded root logger name.